azure/requests.yaml

```diff
@@ -6,6 +6,9 @@ releases:
           issue: https://github.com/giantswarm/giantswarm/issues/13771
     - name: "> 14.1.0"
```

Comment:
Why not > 14.0.0 so it includes 14.0.1? haha

Reply:
Existing releases make this tricky. CI passes now and I think its better.

Reply:
oki!